components/x-live-blog-wrapper/readme.md

```diff
@@ -64,13 +82,59 @@ hydrate();
 ```
 
 ### Live updates
-This component exports a function named `listenToLiveBlogEvents` which is used for listening to client side live blog updates. This function should be called after hydrating the component if it is rendered at the server side. 
+This component exports a function named `listenToLiveBlogEvents` which is used for listening to client side live blog updates.
```

Comment:
Is it worth mentioning here that these come in the form of server sent events sent by next-live-event-api?